andela/ah-frontend-odin
# #161340141 Enable Users to View and Edit their Profiles

#### What does this PR do?
Users should be able to view and edit their profiles
#### Description of Task to be completed?
Users should be able to amend/edit their profiles depending on the change of preference to the full disclosure of the individual user.
##### As a User
I want to have access to my profile.
I want to be able to edit my profile to my preference.
#### How should this be manually tested?
-  Pull branch `ft-user-profile-161340141` and check it out
- Run server with `yarn start` command
- Create an account with the app if you do not have one
- Log into the application
- Click on your username in the navigation bar
- That will lead you to your user profiles page
- Hence you can now test out the edit profiles feature
#### What are the relevant pivotal tracker stories?
(https://www.pivotaltracker.com/story/show/161340141)[#161340141]
#### Screenshots 
<img width="1679" alt="screenshot 2018-12-12 at 10 26 24" src="https://user-images.githubusercontent.com/39129473/49853514-74514500-fdf8-11e8-81ce-b90a316dd2fd.png">
